js/index.js

```diff
@@ -2,41 +2,57 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  const price = product.querySelector('.price span').innerHTML;
```

Comment:
```suggestion
  const price = product.querySelector('.price span').innerText;
```
Siempre que te puedas evitar usar el HTML mejor que mejor! :D